src/js/components/RoutedAnchor/RoutedAnchor.js

```diff
@@ -14,6 +14,11 @@ class RoutedAnchor extends Component {
 
   render() {
     const { path, method, ...rest } = this.props;
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
Is everyone guaranteed to be using `NODE_ENV` in this way? I like having it show up in development only.

Reply:
i think yes. its the same way we do for docs i recall.